content/en/news/releases/1.22.x/announcing-1.22/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing Istio 1.22.0
+linktitle: 1.22.0
+subtitle: Major Release
+description: Istio 1.22 Release Announcement.
+publishdate: 2024-05-13
+release: 1.22.0
+---
+
+We are pleased to announce the release of Istio 1.22 - one of the largest and most impactful releases we've ever launched. Thank you to all our contributors, testers, users and enthusiasts for helping us get the 1.22.0 release published.
+
+We would like to thank the Release Managers for this release, **Jianpeng He** from Tetrate, **Sumit Vij** from Credit Karma and **Zhonghu Xu** from Huawei. Once again, the release managers owe a debt of gratitude to Test & Release WG lead Eric Van Norman for his help and guidance; more on him later.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.22.0 is officially supported on Kubernetes versions `1.27` to `1.30`.
+{{< /tip >}}
+
+## What's new
+
+### Ambient mode now in Beta
+
+Istio’s ambient mode is designed for simplified operations without requiring changes or restarts to your application. It introduces lightweight, shared node proxies and optional Layer 7 per-workload proxies, thus removing the need for traditional sidecars from the data plane. Compared to sidecar mode, ambient mode reduces memory overhead and CPU usage by over 90% in many cases.
+
+Under development since 2022, the Beta release status indicates ambient mode’s features and stability are ready for production workloads with appropriate cautions. [Our ambient mode blog post has all the details](/blog/2024/ambient-reaches-beta/).
```

Comment:
```suggestion
Under development since 2022, the Beta release status indicates ambient mode’s features and stability are ready for production workloads with appropriate precautions. [Our ambient mode blog post has all the details](/blog/2024/ambient-reaches-beta/).
```

Reply:
just to be consistent with the blog